src/codecache/CodeCacheReaderWriter.cpp

```diff
@@ -473,6 +473,20 @@ void CodeCacheWriter::storeByteCodeStream(ByteCodeBlock* block)
                 }
                 break;
             }
+            case ComplexSetObjectOperationOpcode: {
+                ComplexSetObjectOperation* bc = (ComplexSetObjectOperation*)currentCode;
```

Comment:
Could you use a c++ style casting?
https://google.github.io/styleguide/cppguide.html#Casting

Reply:
Good suggestion!
I fixed it to use c++ casting